contrib/cirrus/setup_environment.sh

```diff
@@ -145,7 +145,7 @@ fi
 # Which distribution are we testing on.
 case "$OS_RELEASE_ID" in
     debian)
-        # FIXME 2023-04-11: workaround for runc regression causing failure
+        # FIXME 2023-08-7: workaround for runc regression causing failure
```

Comment:
If you have to repush, could you please revert this change? The important thing here is how old the comment is: that's what will help a future maintainer track the history. You may, if you wish, add a new note "Last checked 2023-08-07 and it's still broken" (please be sure to use leading zeroes).

Reply:
Ahh, yeah I updated it after checking it's still old.